include/lsst/afw/coord/Coord.h

```diff
@@ -131,7 +131,7 @@ class Coord {
     void rotate(Coord const &axis, lsst::afw::geom::Angle const theta);
     lsst::afw::geom::Angle offset(lsst::afw::geom::Angle const phi, lsst::afw::geom::Angle const arcLen);
     
-    Coord::Ptr convert(CoordSystem system) const;
+    Coord::Ptr convert(CoordSystem system, double epoch=2000) const;
```

Comment:
This is a good point, but I would prefer consistency. I can change all instances now or fix it as part of adding getCoordSystem to Coord. I confess I'm inclined to do it as part of the latter work, since it involves more work on Coord anyway.